arch/arm/core/aarch64/switch.S

```diff
@@ -77,12 +64,6 @@ SECTION_FUNC(TEXT, z_arm64_context_switch)
 	ldr	x1, [x2]
 	mov	sp, x1
 
-#ifdef CONFIG_TRACING
```

Comment:
we need to add this back, switched should be traced here and not in c code. see 1dddbecb350265dd9e3387cde5ceb9dd622c5abb

Reply:
I'll put those back.